plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/wizard/NewDataflowProjectWizard.java

```diff
@@ -52,9 +52,9 @@ public boolean performFinish() {
     }
     try {
       getContainer().run(true, true, creator);
-    } catch (InvocationTargetException | InterruptedException e) {
+    } catch (InvocationTargetException | InterruptedException ex) {
       // TODO: handle
-      DataflowUiPlugin.logError(e, "Error encountered when trying to create project");
+      DataflowUiPlugin.logError(ex, "Error encountered when trying to create project");
```

Comment:
I was going to say this:
https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/1579#discussion_r130979346

I didn't really like the rule being applied everywhere blindly. How about us allowing a one-letter var for exceptions?